exercises/scrabble-score/test/scrabble_score_test.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:test/test.dart';
+import 'package:scrabble_score/scrabble_score.dart';
+
+void main() {
+  group('Scrabble score', () {
+    group('should return a score of 0 for', () {
+      test('empty input', () {
+        var word = new ScrabbleScore('');
+        expect(word.getScrabbleScore(), equals(0));
+      }, skip: false);
+    });
+
+    group('should return the appropriate score for', () {
+
+      test('lowercase letters', () {
+        var word = new ScrabbleScore('a');
+        expect(word.getScrabbleScore(), equals(1));
```

Comment:
Just noting that to fit the canonical-data.json's template the tests should look more like:

```dart
    test("lowercase letters", () {
      final int result = scrabbleScore.score("a");
      expect(result, equals(1));
    });
```

No constructor, just a method called score that is passed the input.

Reply:
If that's the case, would it be more proper to utilize Dart's support for high-order functions? No classes, just functions?